File: front-end/src/app/endpoints/kandidat-endpoint/get/kandidat-get-all-response.ts
export interface KandidatGetAllResponse {
  kandidati:{
    $values: KandidatGetAllResponseKandidat[];
    $id: number;
  }
}

export interface KandidatGetAllResponseKandidat {
  id: number
  email: string
  username: string
  ime: string
  prezime: string
  datumRodjenja: string
  mjestoPrebivalista: string
  zvanje: string
  phoneNumber: string
}

File: back-end/JobSearchingWebApp/Endpoints/Kandidat/GetAll/KandidatGetAllResponse.cs
namespace JobSearchingWebApp.Endpoints.Kandidat.GetAll
{
    public class KandidatGetAllResponse
    {
        public List<KandidatiGetAllResponse> Kandidati { get; set; }
    }

    public class KandidatiGetAllResponse
    {
        public string Id { get; set; } 

        public string Email { get; set; }

        public string Username { get; set; }

        public string Ime { get; set; }

        public string Prezime { get; set; }

        public DateTime DatumRodjenja { get; set; }

        public string MjestoPrebivalista { get; set; }

        public string Zvanje { get; set; }

        public string PhoneNumber { get; set; }
    }
}
